src/seismometer/controls/selection.py

```diff
@@ -242,15 +242,12 @@ def __init__(
             selection_value = values[dropdown_value]
             value = (dropdown_value, selection_value)
 
-        self.dropdown = Dropdown(
-            options=[key for key in values],
-            value=value[0],
-            layout=DROPDOWN_LAYOUT,
-        )
+        self.dropdown = Dropdown(options=[key for key in values], value=value[0], layout=DROPDOWN_LAYOUT)
+
         self.dropdown.observe(self._on_selection_change, "value")
         self.selection_widgets = {}
         for key in options:
-            selection_widget = SelectionListWidget(options=options[key], value=values[key])
+            selection_widget = MultiselectDropdownWidget(options=options[key], value=values[key])
```

Comment:
After experimenting with the notebook, I'm not sure this is always an improvement. It might even need some configurability...

The hardest part for me was the ExploreCohortEvaluation widget.  We go out of our way to keep colors aligned with subsets and the legend displays all possibities.  This is a bit jarring when you only see buttons for your selection, and was more... harmonious(?) when the selection aligned.

Reply:
reverted to using the SelectionListWidget for this use case, as its the better experience